inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongUserRoleEntityMapper.java

```diff
@@ -30,6 +30,8 @@ public interface InlongUserRoleEntityMapper {
 
     InlongUserRoleEntity selectById(Integer id);
 
+    InlongUserRoleEntity selectByName(String userName);
```

Comment:
`userName` -> `username`

At the same time, `selectByUsername` may be more accurate to prevent the meaning of `selectByName` from being unclear when adding other fields such as `role_name` and `tenant_name` in the subsequent DB.

Reply:
fixed, thx